.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -58,8 +71,18 @@ body:
       placeholder:
       value:
     validations:
-      required: false
+      required: true
```

Comment:
Sorry, I missed the conversation, why was this changed to true?

Reply:
I think I'd rather leave this as false, not everyone will have an idea how to propose a solution to a problem the've found.